packages/cubejs-materialize-driver/test/MaterializeDriver.test.ts

```diff
@@ -0,0 +1,120 @@
+import { MaterializeDBRunner } from '@cubejs-backend/testing';
+
+import { StartedTestContainer } from 'testcontainers';
+
+import { MaterializeDriver } from '../src';
+
+const streamToArray = require('stream-to-array');
+
+describe('MaterializeDriver', () => {
```

Comment:
Please add here a comment describing the prerequisite shell command to create the materialize db, e.g. `// $ docker run -p 6875:6875 materialize/materialized:v0.25.0 --workers 1`